pkg/cryptoauthlib/include/atca_params.h

```diff
@@ -44,9 +51,19 @@ extern "C" {
  * @{
  */
 
-#ifndef ATCA_PARAM_I2C
-#define ATCA_PARAM_I2C           I2C_DEV(0)
+#ifndef ATCA_PARAM_I2C_DEV0
```

Comment:
There is an other occurrence of `ATCA_PARAM_I2C` in *same54-xpro* does it need change?

Reply:
This will also break external boards, can you add a fallback define